docs/make.bat

```diff
@@ -0,0 +1,36 @@
+@ECHO OFF

```

Comment:
This is for building on Windows? Since we don't support building any other part of Q on Windows, I think we should drop this file.

Reply:
I included windows b/c future contributors may be on Windows.